src/components/TodoTitleField/TodoTitleFiled.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import './TodoTitleField.scss';
+
+type Props = {
+  onTitleChange: (value: string) => void;
+  currentTitle: string;
+};
+
+const TodoTitleField: React.FC<Props> = ({ onTitleChange, currentTitle }) => {
+  const [inputTitle, setInputTitle] = useState(currentTitle);
+
+  const formSubmitHandler = (e: React.FormEvent) => {
+    e.preventDefault();
+    onTitleChange(inputTitle);
+  };
+
+  const onBlur = () => {
+    onTitleChange(inputTitle);
+  };
+
+  const onInput = (e:React.ChangeEvent<HTMLInputElement>) => {
+    setInputTitle(e.target.value);
+  };
+
+  return (
+    <form className="todoTitleField" onSubmit={formSubmitHandler}>
+      <input
+        type="text"
+        className="todoTitleField__input"
```

Comment:
it has the wrong styles (check [the working example](https://mate-academy.github.io/react_todo-app-with-api/))

Reply:
I have not done anything to the styles, I didnt touch them, they came prepacaged from github